README.md

```diff
@@ -10,47 +10,99 @@ master|[![Build Status](https://mseng.visualstudio.com/AzureDevOps/_apis/build/s
 
 Compared to Git's [built-in credential helpers]((https://git-scm.com/book/en/v2/Git-Tools-Credential-Storage)) (Windows: wincred, macOS: osxkeychain, Linux: gnome-keyring) which provides single-factor authentication support working on any HTTP-enabled Git repository, GCM Core provides multi-factor authentication support for [Azure DevOps](https://dev.azure.com/), Azure DevOps Server (formerly Team Foundation Server), and GitHub.
 
-## Public preview for macOS
+## Public preview
 
-The long-term goal of GCM Core is to converge the .NET Framework-based [Git Credential Manager for Windows](https://github.com/Microsoft/Git-Credential-Manager-for-Windows) (GCM Windows), and the Java-based [Git Credential Manager for Mac and Linux](https://github.com/Microsoft/Git-Credential-Manager-for-Mac-and-Linux) (Java GCM), providing a consistent authentication experience across all platforms.
+The long-term goal of Git Credential Manager Core is to converge the .NET Framework-based [Git Credential Manager for Windows](https://github.com/Microsoft/Git-Credential-Manager-for-Windows) (GCM Windows), and the Java-based [Git Credential Manager for Mac and Linux](https://github.com/Microsoft/Git-Credential-Manager-for-Mac-and-Linux) (Java GCM), providing a consistent authentication experience across all platforms.
```

Comment:
maybe define a term here:   "... of Git Credential Manager Core (GCM Core) is to converge ..."

Reply:
I agree that it would be a good idea to define "GCM" as referring to "Git Credential Manager for Windows" and "GCM Core" as "Git Credential Manager".

Reply:
> define "GCM" as referring to "Git Credential Manager for Windows"
I've been trying to avoid "GCM" on it's own referring to "GCM for Windows" as I'm wondering if in the future we might want to drop the "Core" part all together (once the other GCMs are deprecated/archived).